packages/test/stochastic-test-utils/src/markovChain.ts

```diff
@@ -0,0 +1,283 @@
+import { makeRandom } from "./random";
+import { IRandom } from "./types";
+
+abstract class MarkovChain {
+    public static readonly MARKOV_SENTENCE_BEGIN_KEY = "MARKOV_SENTENCE_BEGIN_KEY_01$#@%^#";
+    public static readonly MARKOV_SENTENCE_END_KEY = "MARKOV_SENTENCE_END_KEY_01$#@%^#";
```

Comment:
If you avoid the JSON.parse (see comment below), you could replace these strings with unique Symbols.